lib/modules/manager/sbt/extract.spec.ts

```diff
@@ -378,5 +378,50 @@ describe('modules/manager/sbt/extract', () => {
         packageFileVersion: undefined,
       });
     });
+
+    it('extract sbt version', () => {
+      expect(
+        extract(
+          `sbt.version=1.6.0
+      `,
+          'project/build.properties'
+        )
+      ).toMatchObject({
+        deps: [
+          {
+            datasource: 'github-releases',
+            packageName: 'sbt/sbt',
+            depName: 'sbt/sbt',
+            currentValue: '1.6.0',
+            replaceString: 'sbt.version=1.6.0',
+            versioning: 'semver',
+            extractVersion: '^v(?<version>\\S+)',
+          },
+        ],
+      });
+    });
+
+    it('extract sbt version if the file contains other properties', () => {
+      expect(
+        extract(
+          `sbt.version=1.6.0
```

Comment:
use `codeBlock` string template helper, search codebase for samples. it helps to trim leading spaces